### PR TITLE
Rename master to built-in in German localisation

### PR DESCRIPTION
**Pull request by mPokornyETM**
I am not sure if there are some issues for German localisation. Search for master in jira does not make sense.  But my colleagues by searching built-in node in German language. 
But here is a open question for German community.  How we should names the built-in node in German. Because 'Eingebaute Knote' make from my side no sense.  But my solution is grammary not correct. 

@NotMyFault  what you means?

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

### Proposed changelog entries

* Improve German translation for computer overview

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by NotMyFault — COMMENTED** [review]
These translations are not accurate and do not relate to the original term used. Take a look at the [glossary](https://www.jenkins.io/doc/book/glossary/) how deprecated terms translate in English to provide a base for different translations.

**Comment by basil** [comment]
> These translations […] do not relate to the original term used.

This PR changes `Hudson.Computer.Caption` and `Hudson.Computer.DisplayName` to "Built-in Knote" and "Built-in", respectively. The English versions from `core/src/main/resources/jenkins/model/Messages.properties` are "Built-in Node" and "Built-in", respectively.

I do not have German, but I have logic, and these translations clearly relate to the original term used.

**Comment by basil** [comment]
> They partly relate to them

I am glad you agree that the translations do, in fact, (partly) relate to the original term used.

> though `Hudson.Computer.Caption` and [`Hudson.Computer.DisplayName`] share the same string in the original translation where this proposal diverges from it

Whatever point you are making about `Hudson.Computer.Caption` and `Hudson.Computer.DisplayName` being the same string or different strings, I fail to follow the logic. The translation was done before commit 96a5526bdb1bad3f553d9a9e7f5987a1e6b11456:

```
diff --git a/core/src/main/resources/jenkins/model/Messages.properties b/core/src/main/resources/jenkins/model/Messages.properties
index c7c1d079c7..d5afb130ac 100644
--- a/core/src/main/resources/jenkins/model/Messages.properties
+++ b/core/src/main/resources/jenkins/model/Messages.properties
@@ -24,8 +24,8 @@
 # THE SOFTWARE.
 
 Hudson.BadPortNumber=Bad port number {0}
-Hudson.Computer.Caption=Master
-Hudson.Computer.DisplayName=master
+Hudson.Computer.Caption=Built-in Node
+Hudson.Computer.DisplayName=Built-in
```

The translation was accurate before commit 96a5526bdb1bad3f553d9a9e7f5987a1e6b11456 but has now become out-of-date after commit 96a5526bdb1bad3f553d9a9e7f5987a1e6b11456. Therefore, the translation needs to be be updated. Since commit 96a5526bdb1bad3f553d9a9e7f5987a1e6b11456 made the display name different from the caption, the change to the translation needs to do the same.

File: core/src/main/resources/jenkins/model/Messages_de.properties
Comment (NotMyFault) [inline core/src/main/resources/jenkins/model/Messages_de.properties:25]:
```suggestion
Hudson.Computer.Caption=Eingebauter Knoten
Hudson.Computer.DisplayName=Eingebauter Knoten
```

**Comment by basil** [comment]
The suggestion given in https://github.com/jenkinsci/jenkins/pull/6462#pullrequestreview-938257062 proposes to change the translation of `Hudson.Computer.Caption` and the translation of `Hudson.Computer.DisplayName` to the same string, but this suggestion does not seem cogent to me as explained in https://github.com/jenkinsci/jenkins/pull/6462#issuecomment-1095246735.

**Comment by NotMyFault** [comment]
> The suggestion given in [#6462 (review)](https://github.com/jenkinsci/jenkins/pull/6462#pullrequestreview-938257062) proposes to change the translation of `Hudson.Computer.Caption` and the translation of `Hudson.Computer.DisplayName` to the same string, but this suggestion does not seem cogent to me as explained in [#6462 (comment)](https://github.com/jenkinsci/jenkins/pull/6462#issuecomment-1095246735).

Yeah, typo from my side, oops.

**Review by basil — APPROVED** [review]

File: core/src/main/resources/jenkins/model/Messages_de.properties
Comment (daniel-beck) [reply core/src/main/resources/jenkins/model/Messages_de.properties]: I think the display name is what's shown in the side panel? In English it works with just "built-in" (IMO), less sure about German.

My first thought was to use "integriert" for the built-in equivalent, but that seems wrong without a noun too.

"Interner Knoten" and "Intern" could work?

Comment (NotMyFault) [reply core/src/main/resources/jenkins/model/Messages_de.properties]: `Hudson.Computer.Caption` is used in the `/computer/` overview, "Interner Knoten" sounds fine too.

File: core/src/main/resources/jenkins/model/Messages_de.properties
Comment (NotMyFault) [inline core/src/main/resources/jenkins/model/Messages_de.properties:25]:
```suggestion
Hudson.Computer.Caption=Interner Knoten
Hudson.Computer.DisplayName=Intern
```
Applying Daniel's suggestions

File: core/src/main/resources/jenkins/model/Messages_de.properties
Comment (basil) [reply core/src/main/resources/jenkins/model/Messages_de.properties]: > "Interner Knoten" and "Intern" could work?

I already merged the original suggestion, but if someone offers this new suggestion in the GitHub UI I can accept it as well. I don't have a preference about the specific translation, just that it matches the logical structure of the original (with two different strings).

**Review by NotMyFault — APPROVED** [review]
Reads better now 👏🏻 

Are `localization` labels picked up by the changelog generator? If so, I'd recommend to change the proposed changelog entry to something like `Improve German translation for computer overview` to highlight what actually changed.

**Comment by basil** [comment]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**
